### PR TITLE
feat: update colors

### PR DESCRIPTION
**Pull request by goldenmaya**
### Jira-link:

https://virtocommerce.atlassian.net/browse/VCST-890
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.58.0-pr-1043-7c2f-7c2f90ba.zip

### CODE REVIEW

File: client-app/shared/catalog/components/product-card-grid.vue [hunk 1]
```diff
@@ -141,9 +141,9 @@
             <svg
               class="size-3 shrink-0"
               :class="{
-                'text-[color:var(--color-success)]': true,
-                'text-[color:var(--color-warning)]': false,
-                'text-[color:var(--color-danger)]': false,
+                'text-success': true,
+                'text-warning': false,
```
Comment (NaMax66) [inline client-app/shared/catalog/components/product-card-grid.vue:145]: text-success could be moved to class and other could be removed

File: client-app/shared/catalog/components/product-card-list.vue [hunk 2]
```diff
@@ -78,9 +78,9 @@
           <svg
             class="size-3 shrink-0"
             :class="{
-              'text-[color:var(--color-success)]': true,
-              'text-[color:var(--color-warning)]': false,
-              'text-[color:var(--color-danger)]': false,
+              'text-success': true,
```
Comment (NaMax66) [inline client-app/shared/catalog/components/product-card-list.vue:81]: text-success could be moved to class and other could be removed

File: client-app/shared/wishlists/components/add-to-list.vue [hunk 3]
```diff
@@ -12,7 +12,9 @@
     </template>
 
     <template #content>
-      <div class="max-w-[8rem] rounded-sm bg-white px-3.5 py-1.5 text-xs text-tooltip shadow-sm-x-y xs:max-w-[10rem]">
+      <div
```
Comment (NaMax66) [inline client-app/shared/wishlists/components/add-to-list.vue:15]: Too long. Refactor to BEM

File: client-app/ui-kit/components/molecules/address-form/vc-address-form.vue [hunk 4]
```diff
@@ -52,7 +52,7 @@
       <!-- Divider -->
       <div
         v-if="withPersonalInfo"
-        class="-mx-96 mb-6 mt-8 border-t border-[color:var(--color-primary)] md:mx-9 md:mb-4 md:mt-6 md:border-l"
+        class="-mx-96 mb-6 mt-8 border-t border-primary md:mx-9 md:mb-4 md:mt-6 md:border-l"
```
Comment (NaMax66) [inline client-app/ui-kit/components/molecules/address-form/vc-address-form.vue:55]: to BEM

File: client-app/shared/wishlists/components/add-to-list.vue (hunk 3 above)
Comment (goldenmaya) [reply client-app/shared/wishlists/components/add-to-list.vue:15]: Not here, we have a task for that

File: client-app/ui-kit/components/molecules/address-form/vc-address-form.vue (hunk 4 above)
Comment (goldenmaya) [reply client-app/ui-kit/components/molecules/address-form/vc-address-form.vue:55]: Weird component for ui-kit, I'd like to move this component somewhere to shared components. Need to discuss and create a new task

File: client-app/shared/catalog/components/product-card-grid.vue (hunk 1 above)
Comment (goldenmaya) [reply client-app/shared/catalog/components/product-card-grid.vue]: done

File: client-app/shared/catalog/components/product-card-list.vue (hunk 2 above)
Comment (goldenmaya) [reply client-app/shared/catalog/components/product-card-list.vue]: done

**Review by Andrew-Orlov — APPROVED** [review]

**Pull request merged**